components/dashboard/UpcomingInterviewDialog/UpcomingInterviewValidationRules.js

```diff
@@ -0,0 +1,13 @@
+import isEmpty from 'lodash/isEmpty';
+
+const upcomingInterviewValidation = values => {
```

Comment:
No action here, but food for future thought -- would be cool to get to a point where we can just say `required` on a component (or something like that) and all this happens automatically. As it stands, we're not super DRY with the labels, etc. (but great start to modularizing validation).

Reply:
Totally agree -- there are some lightweight validation rules libraries, are you thinking of starting to pull rules out into their own module or going off of one of those?